### DISCUSSION TITLE
NocoDB n8n integration [Jul 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2815] [Discord Threads]
> 🙋‍♂️ **include**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@309376209647697921> explore here https://docs.n8n.io/integrations/nodes/n8n-nodes-base.nocodb/  
#  
> 💁 **o1lab**  
#### https://github.com/n8n-io/n8n/pull/3146  
#  
> 💁 **o1lab**  
#### please subscribe to this  
#  
> 🙋‍♂️ **include**  
#### cool thanks for sharing 🤙🙏🤌  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/997134230310109264)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
